### PR TITLE
Switch boost::shared_ptr to urdf::*SharedPtr

### PR DESCRIPTION
**Pull request by bhomberg**
It's not compiling on melodic for me without this.

### CODE REVIEW

**Comment by bhomberg** [comment]
@jonbinney , do you want to do a review on this before I merge to the iron_ox repo and then make a PR directly to the fanuc_experimental repo?

File: fanuc_lrmate200id_moveit_plugins/lrmate200id_kinematics/src/fanuc_lrmate200id_manipulator_ikfast_moveit_plugin.cpp
Comment (jonbinney) [inline fanuc_lrmate200id_moveit_plugins/lrmate200id_kinematics/src/fanuc_lrmate200id_manipulator_ikfast_moveit_plugin.cpp]: Don't make any whitespace changes in this PR - if you want, you can put them in a separate PR. Otherwise, just PR the URDF shared ptr stuff to make it super easy for the maintainers to review.

**Review by jonbinney — APPROVED** [review]
Remove whitespace changes. Otherwise good.

File: fanuc_lrmate200id_moveit_plugins/lrmate200id_kinematics/src/fanuc_lrmate200id_manipulator_ikfast_moveit_plugin.cpp
Comment (bhomberg) [reply fanuc_lrmate200id_moveit_plugins/lrmate200id_kinematics/src/fanuc_lrmate200id_manipulator_ikfast_moveit_plugin.cpp]: Fixed; my editor did it automatically...

**Pull request merged**
